org.adempiere.base/src/org/compiere/model/MImportTemplate.java

```diff
@@ -196,30 +203,47 @@ else if ("M".equals(importMode))
 		int cnt = DB.getSQLValueEx(get_TrxName(), sql.toString(), getAD_ImportTemplate_ID(), roleID, roleID);
 		return cnt > 0;
 	}
-	
+
+	/**
+	 * Byte Order Mark character for UTF-8
+	 */
+	public static final String UTF8_BOM = "\uFEFF";
```

Comment:
From my research, this should be the byte order mark for UTF-16BE (Big Endian), not UTF-8. The byte order mark for UTF-8 with BOM should be "EF BB BF" instead. The use of BOM is not recommended by Unicode Standard but it seems Excel is using it.

Reply:
changed name of constant